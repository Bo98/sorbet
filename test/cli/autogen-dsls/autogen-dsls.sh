#!/bin/bash
set -eu

echo "--- autogen-dsls ---"
main/sorbet --silence-dev-message --stop-after=namer -p autogen-dsls \
  test/cli/autogen-dsls/dsls.rb
