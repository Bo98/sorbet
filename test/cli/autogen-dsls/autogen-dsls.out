--- autogen-dsls ---
'::Bar':
  'foo': 5
  'bar': '::SomeOtherClass'
  'baz':
    - "this"
    - "that"
    - "the other"
